apps/omg_api/test/integration/happy_path_test.exs

```diff
@@ -36,16 +36,16 @@ defmodule OMG.API.Integration.HappyPathTest do
     token: token,
     alice_deposits: {deposit_blknum, token_deposit_blknum}
   } do
-    raw_tx = Transaction.new([{deposit_blknum, 0, 0}], eth(), [{bob.addr, 7}, {alice.addr, 3}])
+    raw_tx = Transaction.new([{deposit_blknum, 0, 0}], [{bob.addr, eth(), 7}, {alice.addr, eth(), 3}])
```

Comment:
for the record - the update of `Transaction.new`'s API is OK, this is far less frequently invoked and is not a test helper like `create_encoded` et al.